src/api/helpers/decrypt-financial-uk/index.test.ts

```diff
@@ -33,23 +33,10 @@ describe('api/helpers/decrypt-financial-uk', () => {
     expect(result).toEqual(expected);
   });
 
-  describe('when an empty object is provided', () => {
-    it('should return empty strings for sortCode and accountNumber', () => {
-      const result = decryptFinancialUk({});
-
-      const expected = {
-        sortCode: '',
-        accountNumber: '',
-      };
-
-      expect(result).toEqual(expected);
-    });
-  });
-
```

Comment:
What's the reason for removing this test?

Reply:
by accident, readded